rebasehelper/application.py

```diff
@@ -412,6 +417,23 @@ def patch_sources(self, sources):
         self.rebase_spec_file.write_updated_patches(self.rebased_patches)
         results_store.set_patches_results(self.rebased_patches)
 
+    def generate_patch(self):
+        """
+        Generates patch to the results_dir containing all needed files for
+        the rebased package version
+        """
+        # Copy applied untouched patches to the newsources directory
+        SpecFile.copy_applied_patches(self.new_sources_dir, self.spec_file, self.rebase_spec_file)
+
+        dir = os.getcwd()
+        os.chdir(self.new_sources_dir)
+        gh = GitHelper('./')
+        gh.command_init('./')
+        gh.command_add_files('.')
+        gh.command_commit('Init commit.')
+        gh.command_format_patch(directory = self.results_dir)
```

Comment:
There shoudn't be spaces around `=` when passing a keyword argument, see [PEP 8 style guide](https://www.python.org/dev/peps/pep-0008/#other-recommendations).